md/release-notes.md

```diff
@@ -29,6 +29,7 @@ For more information, read [the dedicated documentation](variables.md).
 
 #### Switch widget
 Directly from whiteboard or from widget properties section, you will now be able to change your current widget by a new one without loosing the configuration you have done. Useful when switching to a new version of your widget custom or not, and as first customization steps after your form creation from studio to match your end user needs.
+⚠️This feature is not available on Edge browser.  
```

Comment:
❓ is this working on IE11?

Reply:
good catch.

Is IE 11 still in the matrix ? i don't have this browser to test, but if same limitation is present, it should also be added of course

Reply:
yes IE11 is in the 7.9 matrix and for now, will also be in the 7.10 matrix
I am asking the UID Team about the IE limitation

Reply:
True, we need to put IE11 limitations too ! Good catch !